onnxscript/function_libs/torch_lib/ops/nn.py

```diff
@@ -2220,7 +2230,7 @@ def _aten_upsample_output_size(
         output_size,
         mode=mode,
         coordinate_transformation_mode=coordinate_transformation_mode,
-        nearest_mode="floor",
+        nearest_mode=nearest_mode,
```

Comment:
```suggestion
        nearest_mode="floor",
```

otherwise we can keep